src/bacnet/datalink/bacsec.h

```diff
@@ -211,10 +211,10 @@ extern "C" {
         uint8_t * msg,
         uint32_t msg_len,
         uint8_t * signature);
-    bool key_verify_sign_msg(BACNET_KEY_ENTRY * key,
-        uint8_t * msg,
-        uint32_t msg_len,
-        uint8_t * signature);
+    // bool key_verify_sign_msg(BACNET_KEY_ENTRY * key,
+    //     uint8_t * msg,
+    //     uint32_t msg_len,
+    //     uint8_t * signature);
```

Comment:
@skarg are you ok with removing those functions from the library? those functions call openssl functions which are not supported anymore and with bacnet secure probably also outdated?

Reply:
Yes, I'm okay with removing the functions that require openssl dependency from the library.